plugins.json

```diff
@@ -80,6 +80,15 @@
         "preferred": false,
         "compatibility": [],
         "url": "https://github.com/spinnaker-plugins/aws-lambda-deployment-plugin-spinnaker/releases/download/1.0.4/aws-lambda-deployment-plugin-spinnaker-1.0.4.zip"
+      },
+      {
+        "version": "1.0.5",
+        "date": "2021-06-09T22:56:43.456161Z",
+        "requires": "orca>=0.0.0,deck>=0.0.0",
```

Comment:
With the latest changes from @nimakaviani  we need to add clouddriver version check as well. 
[fix(stabilize) - verify that lambda state is active ](https://github.com/spinnaker-plugins/aws-lambda-deployment-plugin-spinnaker/pull/76) depend on latest clouddriver changes

Reply:
no not really. this is only for the microservices that the plugin extends and integrates with. we are not directly integrating with clouddriver (not just yet).

Reply:
but 1.05 will not have my changes anyway. we need to merge this and cut 1.0.6 for it to capture changes I made.